pytest_fixtures/certificate.py

```diff
@@ -0,0 +1,150 @@
+from pathlib import Path
+
+import pytest
+from fauxfactory import gen_string
+
+from robottelo import ssh
+from robottelo.config import settings
+from robottelo.helpers import get_data_file
+from robottelo.ssh import get_connection
+from robottelo.ssh import upload_file
+
+
+@pytest.fixture(scope="module")
```

Comment:
This isn't really a fixture, its just a function that you'll call within your other fixture.